Dockerfile.interop_client

```diff
@@ -0,0 +1,17 @@
+# This container image implements the client role of draft-dcook-ppm-dap-interop-test-design-01.
+FROM node:18-alpine
+EXPOSE 8080
+
+RUN mkdir /opt/divviup-ts
+RUN mkdir /logs
+WORKDIR /opt/divviup-ts
+
+COPY package.json .
+RUN npm set progress=false && npm install
```

Comment:
should this be `npm ci` to ensure we use the package-lock.json?

Reply:
Yep, that makes sense. I deleted an erroneous entry in `.gitignore` that had fooled me as well.